services/http_gate/.hosts

```diff
@@ -0,0 +1 @@
+IPV4_PREFIX.81 http_gate.LOCAL_DOMAIN
```

Comment:
Let it be just `http`

Reply:
`IPV4_PREFIX.90 morph_chain.LOCAL_DOMAIN`

double standarts?

Reply:
Almost triple, if `bastion` counts.